src/ui/components/DropdownMenu/index.js

```diff
@@ -34,18 +36,44 @@ export class DropdownMenuBase extends React.Component<Props, State> {
     }));
   }
 
+  handleOnClickForLinks = (event: SyntheticEvent<any>) => {
+    // If a link inside the menu is clicked, we should close the dropdown.
+    // See: https://github.com/mozilla/addons-frontend/issues/3452
+    if (event.target && event.target.tagName === 'A') {
```

Comment:
Are we 100% sure it is a uppercase tag name? I think so but we never know.

Reply:
That's how the DOM API works and React's `SyntheticEvent` normalizes the content across browsers, so it should always be uppercase.